unitTests/ui/CoreHelpersTest.ts

```diff
@@ -218,7 +218,7 @@ export function CoreHelperTest() {
 
         it('should work correctly with useLongDateFormat', () => {
           options.useLongDateFormat = true;
-          expect(TemplateHelpers.getHelper('date')(new Date(1981, 3, 11), options)).toEqual('Saturday, April 11, 1981');
+          expect(TemplateHelpers.getHelper('date')(new Date(1981, 3, 11), options)).toEqual('Saturday April 11 1981');
         });
 
```

Comment:
Should we include a smoke-test for one different language, e.g. french?

Reply:
Hehhh it's a bit of a shame, but with how tests are built/being run, it would be pretty hard.

We do not really support changing the loaded language on the fly, so it seems we'd have to implement that feature first to be able to test different languages more easily.

Otherwise we'd get weird test behaviour where we change the loaded language mid execution, and it would affect all other tests.